Dockerfile

```diff
@@ -91,12 +91,15 @@ RUN true \
   && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
 
 # Do not use root to run the app
-USER node
+# BUT it does not work with secret mount (could not find a solution yet)
```

Comment:
when do we need secret mount?

Reply:
To load Google credentials for bigquery

Reply:
nope, just running as root is good, <i just left the comment to be sure the next person is not trying to put an user back